docs/logging-configuration.asciidoc

```diff
@@ -0,0 +1,105 @@
+ifdef::env-github[]
+NOTE: For the best reading experience,
+please view this documentation at https://www.elastic.co/guide/en/apm/agent/ruby[elastic.co]
+endif::[]
+
+[[logging-configuration]]
+== Logging configuration
```

Comment:
Do we want to call it _Log correlation_ or is that too domain specific? What do the other agents call it?

Reply:
I looked at the [java agent's documentation](https://www.elastic.co/guide/en/apm/agent/java/master/config-logging.html) and saw that all logging documentation was grouped onto one page. I didn't want to repeat the log config options here so I link to them at the top and then document the log correlation below.

One option is to make this page _only_ about log/trace correlation and update the title to _Log correlation_. Would you then like to remove the links to log configuration at the top or keep them as they are indirectly related?

Reply:
I think the two concepts are different enough that it could be easy to mix them up.

The current log config options all have to do with the agent's own logs and where to put them. This new api has to do with how to put the agent into the host app's logs. Therefore I don't think mixing them or putting them too close to each other helps more than it might confuse. Does that make sense?

Reply:
Yes, makes complete sense.

Reply:
https://github.com/elastic/apm-agent-ruby/pull/527/commits/cd8be8b51d9fbdde29c04d999eef1fc3644be28b